.github/workflows/design-system-governance.yml

```
@@ -0,0 +1,523 @@
+    
+    steps:
+      - name: 📥 Checkout Code
+        uses: actions/checkout@v4
+        
+      - name: 🛡️ Dependency Security Audit
+        run: |
+          echo "🔍 Running security audit..."
+          npm audit --audit-level=moderate
+          
+      - name: 🕵️ Vulnerability Scanning
+        uses: github/codeql-action/analyze@v2
```

actionlint (1.7.7)

the runner of "github/codeql-action/analyze@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)